src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -217,8 +222,22 @@ public void start(final CommandMap commandMap)
                 CommonConstants.COMMA);
         final MapRouletteConfiguration mapRouletteConfiguration = (MapRouletteConfiguration) commandMap
                 .get(MAP_ROULETTE);
-        final Configuration checksConfiguration = ConfigurationResolver
-                .loadConfiguration(commandMap, CONFIGURATION_FILES, CONFIGURATION_JSON);
+        @SuppressWarnings("unchecked")
+        final Optional<List<String>> checkFilter = (Optional<List<String>>) commandMap
+                .getOption(CHECK_FILTER);
+
+        final Configuration checksConfiguration = new MergedConfiguration(Stream
+                .concat(Stream.of(
+                        ConfigurationResolver.loadConfiguration(commandMap, CONFIGURATION_FILES,
+                                CONFIGURATION_JSON)),
+                        Stream.of(checkFilter
+                                .<Configuration> map(whitelist -> new StandardConfiguration(
```

Comment:
Whats with the casting here??

Reply:
Otherwise, `checkFilter.map(whitelist -> new StandardConfiguration(...))` returns an `Optional<StandardConfiguration>`. So then the `orElse` would need to take a `StandardConfiguration`, and I didn't see a concise way to create an empty `StandardConfiguration`. 

There might be a better way around this, but this was the cleanest way I saw to keep everything concise.